### DISCUSSION TITLE
Adding sniffing NIC after install?

### DISCUSSION THREAD

**Discussion by lukepenick** [Security-Onion-Solutions/securityonion#1903] [Unsupported Versions] [answered]
I’ve looked up and down in the documentation and can’t find much on this. (I may be over looking) 

Set up 2.3.2 in proxmox via network install on Ubuntu 18.04. 

Is there a way to add in a sniffing NIC after the install? I have added more NICs and don’t really know what to do now...

**Comment by TOoSmOotH** [comment, marked as answer]
Just add the new NICs to bond0 and they will get monitored.

**Reply by n3kt0n** [reply]
This should definitely be in the FAQ.  Seems like this would be a very common task for organizations with expanding visibility requirements.  I dug through configuration for the initial sniff interface hoping to duplicate that for the new one I just added, but it was not clear which things I would need to change.  In hindsight, adding to bond0 makes sense.  But not then.  So I re-ran the most obvious solution I could find "so-setup-network" and it basically reinstalled Security Onion on top of the old one. It is still running. Hopefully it is not installing a parallel version

**Reply by n3kt0n** [reply]
And, looks like I will have to rebuild the stack
